Localization/localization.cfg

```diff
@@ -131,135 +131,135 @@ Localization
+                
+        precisemaneuver_mainmenu_enable                 = Habilitar
+        precisemaneuver_mainmenu_keybindings            = Establecer atajos
+        precisemaneuver_mainmenu_background             = Fondo
+        precisemaneuver_mainmenu_scale                  = Scala de la GUI
+        precisemaneuver_mainmenu_components             = COMPONENTES
+                
+        precisemaneuver_module_pager                    = Buscador de Maniobras
+        precisemaneuver_module_saver                    = Plantillas de Maniobras
+        precisemaneuver_module_precise_input            = Entrada precisa
+        precisemaneuver_module_orbit_tools              = Herramientas de Órbita
+        precisemaneuver_module_gizmo                    = Gadget de Maniobra
+        precisemaneuver_module_next_encounter           = Siguiente encuentro
+        precisemaneuver_module_ejection                 = Ángulo de eyeción
+        precisemaneuver_module_info                     = Informacion orbital
+        precisemaneuver_module_patches                  = Control de parches
+                
+        precisemaneuver_keybindings_set                 = Activar
+        precisemaneuver_keybindings_unset               = Desactivar
+                
+        precisemaneuver_keybindings_enable              = Mostrar/ocultar ventana
+        precisemaneuver_keybindings_proginc             = Incremenar progrado
+        precisemaneuver_keybindings_progdec             = Reducir progrado
+        precisemaneuver_keybindings_progzero            = Anular progrado
+        precisemaneuver_keybindings_norminc             = Incrementar normal
+        precisemaneuver_keybindings_normdec             = Reducir normal
+        precisemaneuver_keybindings_normzero            = Anular normal
+        precisemaneuver_keybindings_radinc              = Incrementar radial
+        precisemaneuver_keybindings_raddec              = Reducir radial
+        precisemaneuver_keybindings_radzero             = Anular radial
+        precisemaneuver_keybindings_timeinc             = Incrementar tiempo
+        precisemaneuver_keybindings_timedex             = Reducir tiempo
+        precisemaneuver_keybindings_circularize         = Circularizar órbita
+        precisemaneuver_keybindings_turnup              = Agregar una órbita
+        precisemaneuver_keybindings_turndown            = Reducir una órbita
+        precisemaneuver_keybindings_pageinc             = Cambiar incremento (w/Alt)
+        precisemaneuver_keybindings_pageconics          = Cambiar modo de órbita (w/Alt)
+        precisemaneuver_keybindings_encfocus            = Enfocar siguiente encuentro
+        precisemaneuver_keybindings_patchesmore         = Mostrar más parches
+        precisemaneuver_keybindings_patchesless         = Mostrar menos parches
+        precisemaneuver_keybindings_timex10             = Cambiar x10 veces
```

Comment:
It means toggle x10 FOR time, means, toggle the increment being ten times bigger for UT.